nimble/host/src/ble_l2cap.c

```diff
@@ -387,6 +387,23 @@ ble_l2cap_rx(struct ble_hs_conn *conn,
             goto err;
         }
 
+        if ((chan->dcid >= 0x0040 && chan->dcid <= 0x007F && l2cap_hdr.len > chan->my_coc_mps) ||
```

Comment:
@KKopyscinski  don't you want just add `ble_l2cap_disconnect(chan);` after line 413 ?